content/sensu-core/2.0/reference/agent.md

```diff
@@ -0,0 +1,822 @@
+- [Keepalive monitoring](#keepalive-monitoring)
+- [Operation](#operation)
+	- [Starting the service](#starting-the-service)
+	- [Service management](#stopping-the-service)
+	- [Registration and deregistration](#registration)
+	- [Clustering](#clustering)
+- [Configuration](#configuration)
+  - [API configuration](#api-configuration-flags)
+  - [Ephemeral agent configuration](#ephemeral-agent-configuration-flags)
+  - [Keepalive configuration](#keepalive-configuration-flags)
+  - [Security configuration](#security-configuration-flags)
+  - [Socket configuration](#socket-configuration-flags)
+  - [StatsD configuration](#statsd-configuration-flags)
+
+The Sensu agent is a lightweight client that runs on the infrastructure components you want to monitor.
+Agents are responsible for creating [check and metrics events][7] to send to the [backend event pipeline][2].
+
+_NOTE: The commands in this reference may require administrative privileges or use of `sudo`._
+
+## Creating events using service checks
+
+Sensu's use of the [publish/subscribe pattern of communication][15] allows for automated registration and deregistration of ephemeral systems.
+At the core of this model are Sensu agent subscriptions.
+
+Each Sensu agent has a defined set of [`subscriptions`][28], a list of roles and responsibilities assigned to the system (for example: a webserver or database).
+These subscriptions determine which [monitoring checks][14] are executed by the agent.
+Agent subscriptions allow Sensu to request check executions on a group of systems at a time, instead of a traditional 1:1 mapping of configured hosts to monitoring checks.
+Sensu checks target Sensu agent subscriptions, using the [check definition attribute `subscriptions`][14].
+
+After receiving the check request from the Sensu backend, the agent:
+
+1. Applies any [tokens][27] matching attribute values in the check definition.
+2. Fetches [assets][29] and stores them in its local cache. By default, agents cache asset data at `/var/cache/sensu/sensu-agent` (`C:\\ProgramData\sensu\cache\sensu-agent` on Windows systems) or as specified by the the [`--cache-dir` flag][30].
+3. Executes the [check `command`][14].
+4. Executes any [hooks][31] specified by the check based on the exit status.
+5. Creates an [event][7] containing information about the applicable entity, check, and metric.
+
+### Subscription configuration
+
+In order for an agent to execute a service check, you must specify the same subscription in the [agent configuration][24] and the [check configuration][32].
+To configure subscriptions for an agent, see [the `subscriptions` flag reference][28].
+
+In addition to the subscriptions defined in the agent configuration, Sensu agents also subscribe automatically to a subscription matching their agent `id`.
+For example, an agent with the ID `i-424242` will subscribe to check requests with the subscription `agent:i-424242`.
```

Comment:
@palourde This is great feedback! I've provided more clarification for this in https://github.com/sensu/sensu-docs/pull/824/commits/5d5386092ef48a9f133e0ae3c37671d0f70486ae Let me know what you think.

My goal is to help users understand that their agents will appear as entities within Sensu, but that they need to configure subscriptions in the agent configuration instead of the entity definition.